src/app/zap-templates/zcl/data-model/chip/push-av-stream-transport-cluster.xml

```diff
@@ -0,0 +1,229 @@
+    <item fieldId="3" name="EndpointID" type="int16u"/>
+    <item fieldId="4" name="URL" type="long_char_string" length="256"/>
+    <item fieldId="5" name="TriggerOptions" type="TransportTriggerOptionsStruct"/>
+    <item fieldId="6" name="IngestMethod" type="IngestMethodsEnum" min="0x00" max="0x00"/>
+    <item fieldId="7" name="ContainerFormat" type="ContainerFormatEnum" min="0x00" max="0x00"/>
+    <item fieldId="8" name="ContainerOptions" type="ContainerOptionsStruct"/>
+    <item fieldId="9" name="MetadataOptions" type="MetadataOptionsStruct" isNullable="true" optional="true"/>
+    <item fieldId="10" name="ExpiryTime" type="epoch_s" isNullable="true" optional="true"/>
+  </struct>
+
+  <struct name="TransportTriggerOptionsStruct" apiMaturity="provisional">
+    <cluster code="0x0555"/>
+    <item fieldId="0" name="TriggerType" type="TransportTriggerTypeEnum" min="0x00" max="0x02"/>
+    <item fieldId="1" name="MotionZones" array="true" type="TransportZoneOptionsStruct" isNullable="true" optional="true"/>
+    <item fieldId="2" name="MotionSensitivity" type="int8u" isNullable="true" optional="true" default="5" min="1" max="10"/>
+    <item fieldId="3" name="MotionTimeControl" type="TransportMotionTriggerTimeControlStruct" optional="true"/>
+    <item fieldId="4" name="MaxPreRollLen" type="int16u" optional="true"/>
+  </struct>
+
+  <struct name="TransportZoneOptionsStruct" apiMaturity="provisional">
+    <cluster code="0x0555"/>
+    <item fieldId="1" name="Zone" type="int16u" isNullable="true"/>
+    <item fieldId="2" name="Sensitivity" type="int8u" optional="true" default="5" min="1" max="10"/>
+  </struct>
+
+  <cluster>
+    <domain name="Cameras"/>
+    <name>Push AV Stream Transport</name>
+    <code>0x0555</code>
+    <define>PUSH_AV_STREAM_TRANSPORT_CLUSTER</define>
+    <description>This cluster implements the upload of Audio and Video streams from the Camera AV Stream Management Cluster using suitable push-based transports.</description>
+    <client init="false" tick="false">true</client>
+    <features>
+      <feature bit="0" code="PERZONESENS" name="PerZoneSensitivity" summary="Supports a sensitivity value per Zone">
+        <optionalConform/>
+      </feature>
+    </features>
+    <server init="false" tick="false">true</server>
+    <globalAttribute code="0xFFFD" side="either" value="1"/>
+    <attribute code="0x0000" side="server" define="SUPPORTED_CONTAINER_FORMATS" type="SupportedContainerFormatsBitmap" min="0x00" max="0x01">SupportedContainerFormats</attribute>
+    <attribute code="0x0001" side="server" define="SUPPORTED_INGEST_METHODS" type="SupportedIngestMethodsBitmap" min="0x00" max="0x01">SupportedIngestMethods</attribute>
+    <attribute code="0x0002" side="server" define="CURRENT_CONNECTIONS" type="array" entryType="int16u">CurrentConnections</attribute>
+    <command code="0x00" source="client" name="AllocatePushTransport" optional="false" response="AllocatePushTransportResponse">
```

Comment:
It has `L` in the spec because URL's can be very large and we don't want non Large Transport Controllers to interact with the commands in this cluster as the underlying transport requires TLS/TCP anyway.

Reply:
@aronrosenberg The URL in this case says "max 256" for the size, no?

The issue I see is not whether controllers that don't support L can invoke these commands; it's whether controllers that _do_ support L have to establish a TCP session to invoke these commands.  Why should they, if the payloads in fact fit in a UDP packet?

Reply:
That 256 limit needs to be changed to 2000 as that is the defacto limit in HTTP. (2048 for most browsers, but CMAF appends chunk names after the publishing point. Will change in the spec.

Reply:
OK, that makes a lot more sense in terms of needing L, then.